src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -15,9 +16,14 @@ const idt2Markup = `<div>IDT 2 Markup</div><script type="text/javascript" src="l
 
 const idt1Markup = `<div>IDT 1 Markup</div><script type="text/javascript" src="localhost/idt1.js"></script>`;
 
+const canonicalPathname = 'https://www.bbc.com/service/foo';
+
+const ampPathname = 'https://www.bbc.com/service/foo.amp';
+
 describe('convertInclude', () => {
   const includesBaseUrl = 'https://foobar.com/includes';
   process.env.SIMORGH_INCLUDES_BASE_URL = includesBaseUrl;
+  process.env.SIMORGH_INCLUDES_BASE_AMP_URL = `${includesBaseUrl}/amp`;
```

Comment:
```suggestion
  process.env.SIMORGH_INCLUDES_BASE_AMP_URL = `https://foobar-amp.com/includes`;
```
To fix the stange path issue

Reply:
Also these `process.env` vars may want clearing up as mentioned here: https://github.com/bbc/simorgh/pull/6873/files#r441534318